source/class/qx/ui/form/FileSelectorButton.js

```diff
@@ -146,6 +157,7 @@ qx.Class.define("qx.ui.form.FileSelectorButton", {
       );
       input.addListenerOnce("appear", e => {
         let inputEl = input.getDomElement();
+        input.setStyle('display', 'none');
```

Comment:
```suggestion
       // since qx.html.Node does not even create the
       // domNode if it is not set to visible initially
       // we have to quickly hide it after creation.
       input.setVisible(false);
```

Reply:
@oetiker Is there a `visible` property with boolean value? Do you mean `input.setVisiibility("excluded")`? (Equivalent to `input.exclude()`).

From Widget.js:

```
    /**
     * Make this widget visible.
     *
     */
    show() {
      this.setVisibility("visible");
    },

    /**
     * Hide this widget.
     *
     */
    hide() {
      this.setVisibility("hidden");
    },

    /**
     * Hide this widget and exclude it from the underlying layout.
     *
     */
    exclude() {
      this.setVisibility("excluded");
    },
```